package.json

```diff
@@ -14,9 +14,13 @@
     "grunt-sass": "^1.0.0",
     "grunt-contrib-less": "1.3.0",
     "grunt-less-to-sass": "0.0.10",
-    "jit-grunt": "^0.9.1"
+    "jit-grunt": "^0.9.1",
+    "swagger-ui": "^2.2.5"
   },
   "devDependencies": {
     "grunt-contrib-watch": "~0.6.1"
+  },
+  "scripts": {
+    "postinstall": "rm -rf public/swagger-ui; cp -r node_modules/swagger-ui/dist public/swagger-ui; sed -i .orig \"s/defaultModelRendering: 'schema'/defaultModelRendering: 'model'/\" public/swagger-ui/index.html; sed -i .orig 's/url = \".*\"/url = \"..\\/..\\/api\\/v1\\?swagger\"/' public/swagger-ui/index.html"
```

Comment:
We should update the "package" task in build.xml to make the appropriate calls to install this stuff and to ensure that the node_modules directory or other non-essential Node.js-specific code stays out of the distribution packages at release time. I can help with this if you're not already familiar with Phing -- just put a TODO item in the top-level description and let me know and I'll take care of it.